test.js

```diff
@@ -40,3 +41,31 @@ describe('updateNotifier', function () {
 		}));
 	});
 });
+
+describe('updateNotifier with fs error', function () {
+	before(function () {
+		clearRequire('./');
```

Comment:
@SBoudrias Right.

Unfortunately I couldn't think of a better way to get `xdg-basedir` to update its `.config` value which is used by `configstore`. Had to clear the requires for all three to force the reload of dependencies down the tree.

I'm certainly open to suggestions. :)